tools/shared/pragma.c

```diff
@@ -655,16 +655,14 @@ do_sw(void)
       if(craydir) {
         typ = gtok();
         if (typ != T_IDENT) {
-          backup_nowarn = gbl.nowarn;
-          gbl.nowarn = false;
-          errwarn((error_code_t)602);
-          gbl.nowarn = backup_nowarn;
+          bclr(DIR_OFFSET(currdir, x[19]), 0x18);
+          bset(DIR_OFFSET(currdir, x[19]), 0x400);
           break;
         }
         LCASE(ctok);
         if (strcmp(ctok, "always") == 0) {
           bclr(DIR_OFFSET(currdir, x[19]), 0x18);
-          bset(DIR_OFFSET(currdir, x[19]), 0x400);
+          bset(DIR_OFFSET(currdir, x[191]), 0x2);
```

Comment:
Why was a change required?

Reply:
Not required, per say, but I did that so as to not alter any existing work.
If desired, I shall use the same xflag/xbit values.

Reply:
This block is not executed if `craydir` is false. I don't think it gets set to true by `!dir$ vector`? Why isn't the same change needed on line 674?

Reply:
The change was not required on line 674 because `191, 0x2` is exclusively for vector always.
Everywhere else, `19, 0x400` is used since that is the one for `!dir$ vector`.
Since line 674 deals with `!dir$ vector` alone, `191, 0x2` is not used there.

Unless, you want me to call `bclr()` on that bit at/near line `674`?